src/App.js

```diff
@@ -1,10 +1,194 @@
+import { useState } from 'react';
+import axios from 'axios';
+import Swal from 'sweetalert2';
+import withReactContent from 'sweetalert2-react-content'
+
+// components
+import UserOptions from './Components/UserOptions';
+
+// stylesheet
+import './styles/sass/App.scss'
+
+const MySwal = withReactContent(Swal)
 
 function App() {
+  const [boredResponse, setBoredResponse] = useState('');
+  const [showForm, setShowForm] = useState(true);
+  const [userTypeChoice, setUserTypeChoice] = useState('');
+  const [userPriceChoice, setUserPriceChoice] = useState('');
+  const [userNumberChoice, setUserNumberChoice] = useState('');
+  const [disableNumberMenu, setDisableNumberMenu] = useState(false);
+  const [showStyling, setShowStyling] = useState(false);
+
+  const handleUserTypeChoice = (e) => {
```

Comment:
I think moving the Form logic to the separate component might be a good choice for you, as the app would be even a bit easier to read (and to develop further, if you'd want to add some additional functionality).